### DISCUSSION TITLE
my output graph is not shown

### DISCUSSION THREAD

**Discussion by HamzaAli195** [AammarTufail/python_ka_chilla2023#99] [Q&A]
My problem is that my graph code runs successfully but graph does not shown(I used seaborn for it). I search it on google but don't find answer yet. I think its vs code setting problem that don't allow to show graphs(I am not confident its vs code problem).
problem pic is below.
Need guidance, anyone knows about it?
Jazak Allah

![fig_error](https://user-images.githubusercontent.com/121845773/216511673-cfe7436b-b9c9-46d2-a110-65f21ec03d60.jpeg)

**Comment by Muhammad-Ali-Butt** [comment]
Try this and let me know if it worked or not `Import matplotlib.pyplot as plt
#Then after the countplot code write:
plt.show()`

Also try making plot with matplotlib and see if works or not

**Reply by shahbaz-prog** [reply]
hamza click on the </> sign before the <figure size. It will then give you option to view the graph in different format.
Jazak Allah
